man/less-variables.xml

```diff
@@ -26,7 +26,11 @@
       <term><varname>$SYSTEMD_LESS</varname></term>
 
       <listitem><para>Override the options passed to <command>less</command> (by default
-      <literal>FRSXMK</literal>).</para></listitem>
+      <literal>FRSXMK</literal>).</para>
+
+      <para>If the value of <varname>$SYSTEMD_LESS</varname> does not include <literal>K</literal>,
+      and the pager that is invoked is <command>less</command>, ^C will be ignored by the executable.
+      This allows <command>less</command> to handle ^C itself.</para></listitem>
```

Comment:
i wonder if ^C isn't a bit too nerdy here, maybe Ctrl-C would be more descriptive... Dunno.